### PR TITLE
Add wgDiscordAvatarUrl to ManageWikiSettings

### PR DESCRIPTION
**Pull request by Kelvs599**

### CODE REVIEW

File: ManageWikiSettings.php [hunk 1]
```diff
@@ -1187,6 +1187,15 @@
 		'help' => 'This allows to set whether or not it should be possible for anonymous visitors of the wiki to dismiss the site notice shown.',
 		'requires' => [],
 	],
+	'wgDiscordAvatarUrl' => [
+		'name' => 'Discord Notification Avatar URL',
+		'from' => 'mediawiki',
+		'type' => 'text',
+		'overridedefault' => false,
```
Comment (Universal-Omega) [inline ManageWikiSettings.php:1194]: Should be '' to match  LS I think.  Not a huge deal just what I think it should be.

File: ManageWikiSettings.php [hunk 2]
```diff
@@ -1187,6 +1187,15 @@
 		'help' => 'This allows to set whether or not it should be possible for anonymous visitors of the wiki to dismiss the site notice shown.',
 		'requires' => [],
 	],
+	'wgDiscordAvatarUrl' => [
+		'name' => 'Discord Notification Avatar URL',
+		'from' => 'mediawiki',
+		'type' => 'text',
```
Comment (Universal-Omega) [inline ManageWikiSettings.php:1193]: Why not use 'type' => 'url'?

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: ManageWikiSettings.php (hunk 1 above)
Comment (Kelvs599) [reply ManageWikiSettings.php]: Fixed it.

**Review by Universal-Omega — APPROVED** [review]

File: ManageWikiSettings.php (hunk 1 above)
Comment (Universal-Omega) [reply ManageWikiSettings.php]: Thanks!

File: ManageWikiSettings.php (hunk 2 above)
Comment (Kelvs599) [reply ManageWikiSettings.php]: Sorry, I was copying wgLogo and didn't realize.

Comment (Universal-Omega) [reply ManageWikiSettings.php]: That's alright, thanks!

**Pull request merged**